src/styl/responsive/mobile.styl

```diff
@@ -0,0 +1,37 @@
+// HEADER
+
+.header-title
```

Comment:
header title seems off semantically. as it is applied to menu items as well as title items